cads_adaptors/adaptors/mars.py

```diff
@@ -12,6 +12,10 @@ def convert_format(
     context: Context,
     **kwargs,
 ) -> list:
+    if isinstance(data_format, (list, tuple)):
+        assert len(data_format) == 0, "Only one value of data_format is allowed"
```

Comment:
Should this be 1? At the moment if the assert passes, surely the following line will always fail
```suggestion
        assert len(data_format) == 1 "Only one value of data_format is allowed"
```